### DISCUSSION TITLE
[Suggestion] Use Vim License

### DISCUSSION THREAD

**Discussion by blacklightpy** [helix-editor/helix#9782] [Ideas] [answered]
Vim is Charityware.

To upkeep the tradition set by the author of Vim, I suggest Helix Editor too use the same license.

That is one of the factors that put me off while moving to Helix.

**Comment by blacklightpy** [comment]
Oops I realize the developer may need to earn from this project, which makes it incompatible with this license. However I'll leave it open to give it a thought.

**Comment by kirawi** [comment]
About the choice of license: https://matrix.to/#/!zMuVRxoqjyxyjSEBXc:matrix.org/$MO0AqKtL7MBXvHdImgrvZQDF5f-K88ZYWbVqApF34q8?via=matrix.org&via=mozilla.org&via=tchncs.de

**Comment by archseer** [comment, marked as answer]
We'll stick with our current license but that shouldn't stop you from donating to charity.
